tests/rpmbuild.at

```diff
@@ -2413,6 +2414,7 @@ runroot rpm -q --provides -p /build/RPMS/noarch/bcondtest-1.0-1.noarch.rpm |
 ],
 [])
 
+
```

Comment:
Couple of unrelated empty lines getting added here and above, and also just before and between the new tests. It's a good idea to eyeball 'git diff' and/or 'git show' output for irregularities like this before pushing.

Other than that, looks fine to me now.